types/tx.go

```diff
@@ -77,6 +77,27 @@ func (txs Txs) SplitIntoShares() NamespacedShares {
 	return shares
 }
 
+// ToSliceOfBytes converts a Txs to slice of byte slices.
+//
+// NOTE: This method should become obsolete once Txs is switched to [][]byte.
+// ref: #2603
```

Comment:
Link to actual issue instead of just a number

Reply:
link added https://github.com/celestiaorg/celestia-core/pull/631/commits/48e9ebfe23afd28b8bbedb9a77c911698087e325

fwiw, this was inherited from one of the cherry picked PRs tho, and has likely been there for a long time. Agree that we should definitely add links where possible tho